crane/config.py

```diff
@@ -87,6 +89,8 @@ def read_config(app, parser):
             with supress(NoOptionError):
                 app.config[key] = int(parser.get(SECTION_GENERAL, key))
 
+    app.config['DEBUG'] = app.config.get('DEBUG') or os.environ.get(DEBUG_ENV_NAME) == 'true'
```

Comment:
It might be friendly to throw a .lower() on this so that True or TRuE will work.